src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+    "NETStandard.Library": "1.6.0",
+    "Newtonsoft.Json": "7.0.1",
+    "NuGet.Versioning": "3.5.0-beta2-1456"
```

Comment:
This came from the buildtools project.json, and it's what's available on [https://dotnet.myget.org/feed/dotnet-buildtools/package/nuget/NuGet.Versioning](https://dotnet.myget.org/feed/dotnet-buildtools/package/nuget/NuGet.Versioning). We can push a new version if an update seems necessary.

---
In reply to: [69943654](https://github.com/dotnet/buildtools/pull/863#discussion_r69943654) [](ancestors = 69943654)